orbit/pkg/update/runner.go

```diff
@@ -290,9 +296,16 @@ func (r *Runner) updateTarget(target string) error {
 	}
 	path := localTarget.ExecPath
 
+	if target == "osqueryd" {
+		// Compare old/new osquery versions
+		_ = compareVersion(path, r.OsqueryVersion, "osquery")
+	}
+
 	if target != "orbit" {
 		return nil
 	}
+	// Compare old/new orbit versions
+	_ = compareVersion(path, build.Version, "fleetd")
```

Comment:
`fleetd` is the whole package (orbit + osqueryd + Fleet Desktop). This should be `"orbit"`.

Reply:
I'm following the recent guidance to stop using "orbit": https://fleetdm.slack.com/archives/C019FNQPA23/p1698170586413309

Currently, `fleetd` means the whole package as well as the "orbit" part of that package.

Reply:
Correct. But behind the scenes (actual source code) there's an orbit executable/component/target.

Reply:
Even if we don't want to use `orbit` in user docs, `orbit` is still the main component and executable of fleetd, it is a target of the TUF repository (built and updated when we release new versions of fleetd).

Let me know if this doesn't make sense.

Reply:
Business guidance is to use `fleetd`: https://fleetdm.slack.com/archives/C019FNQPA23/p1704791234553829?thread_ts=1698170586.413309&cid=C019FNQPA23

I think eventually we want to rename `orbit` to `fleetd` and call the current `fleetd` something else, like Fleet Agent.

FYI: @sharon-fdm

Reply:
I am ok to leave this as "fleetd" if everyone thinks so.

It's just that we are printing that there's a downgrade in the "orbit" component of fleetd and showing such component as "fleetd".

Reply:
> I think eventually we want to rename orbit to fleetd and call the current fleetd something else, like Fleet Agent.

Agree, and at that point it does make sense to change this from "orbit" to "fleetd". But anyways, this is no big deal as it's a log and we know in this case that "fleetd" will mean "orbit".